Lab2b/syntax.txt

```diff
@@ -0,0 +1,22 @@
+Instructions are separated by ";"
+<type> ::= int|char|string
+<decl> ::= <type> <identifier>;
+<arraydecl> ::= <type> <identifier>[<no>];
```

Comment:
Is it ok to declare an array with size 0?

Reply:
Yes, from a syntactic point of view